cclib/method/ddec.py

```diff
@@ -271,27 +272,94 @@ def calculate(self, indices=None, fupdate=0.05):
             self.radial_grid_r.append(r)
 
         # Carry out step 3 of DDEC6 algorithm [Determine conditioned charge density and tau]
-        self.logger.info("Conditioning charge densities.")
+        self.logger.info("Conditioning charge densities. (Step 3/7)")
         self.condition_densities()
 
+        # STEP 4-7
+        # Optimize weights on each grid point
```

Comment:
It would be great if you could also add some flavor for how steps 4-7 are different, if at all. Unless you decide to go with the unrolled form, there just comment above each step.

Reply:
I've added some details prior to each step. Thanks.